### DISCUSSION TITLE
conversion of hkl uvw to orientation (Bunge)

### DISCUSSION THREAD

**Discussion by rajesh-khatirkar** [mtex-toolbox/mtex#1112] [Ask Anything]
Dear All,

I am using orientation('Miller',[h k l],[u v w],ss,cs) command to convert hkl uvw to orientation. It works fine when hkl uvw are perfectly orthogonal.

I did some simulations and this orthogonality is slightly lost. I mean its not perfect. Is there a way round for this.  My simulation give me hkl uvw and since its deformation simulation, orthogonality is slightly lost.

I did reverse by using round2Miller(orientation) from my EBSD data. Now I want to get back my orientations for further processing.

Thanks in advance.

Rajesh

**Comment by kilir** [comment]
Hi,
interesting to deform a crystal lattice in a way it loses it's symmetry.

If you feel comfortable about it, you can change the threshold in line 67 in `orientation/byMiller.m` here https://github.com/mtex-toolbox/mtex/blob/61a2bf70f348eac8398e2c6b743e8159d62100bd/geometry/%40orientation/byMiller.m#L67
however be aware that you orientations might not be correct.

Cheers,
Rüdiger

**Reply by rajesh-khatirkar** [reply]
Hi Rüdiger,

Thank for reply. I will do the necessary changes.

However, i dont think that the symmetry is lost during deformation. It is possibly measurement error. There will be lot of misorientations after deformation making the exact measurement very difficult.

Thanks once again.

Rajesh
